_build/build.php

```diff
@@ -5,39 +5,52 @@
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Process\Process;
-use Symfony\Component\Process\Exception\ProcessFailedException;
 use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use SymfonyDocsBuilder\BuildConfig;
+use SymfonyDocsBuilder\DocBuilder;
 
 (new Application('Symfony Docs Builder', '1.0'))
     ->register('build-docs')
     ->addOption('generate-fjson-files', null, InputOption::VALUE_NONE, 'Use this option to generate docs both in HTML and JSON formats')
     ->addOption('disable-cache', null, InputOption::VALUE_NONE, 'Use this option to force a full regeneration of all doc contents')
     ->setCode(function(InputInterface $input, OutputInterface $output) {
-        $command = [
-            'php',
-            'vendor/symfony/docs-builder/bin/console',
-            'build:docs',
-            sprintf('--save-errors=%s', __DIR__.'/logs.txt'),
-            __DIR__.'/../',
-            __DIR__.'/output/',
-        ];
-
-        if ($input->getOption('generate-fjson-files')) {
-            $command[] = '--output-json';
+        $io = new SymfonyStyle($input, $output);
+        $io->text('Building all Symfony Docs...');
+
+        $outputDir = __DIR__.'/output';
+        $buildConfig = (new BuildConfig())
+            ->setSymfonyVersion('4.4')
+            ->setContentDir(__DIR__.'/..')
+            ->setOutputDir($outputDir)
+            ->setImagesDir(__DIR__.'/output/_images')
+            ->setImagesPublicPrefix('_images')
+            ->setTheme('sfdocs')
```

Comment:
This must be `rtd`, the sfdocs theme doesn't exist outside the symfony.com repo.

Reply:
You are right ... fixed in https://github.com/symfony/symfony-docs/commit/b043aa36407257a540c8dbbc3f08ab405429d225 Thanks!